src/settings/components/SaveButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  omit,
+} from 'lodash';
+
+import { Button } from '@folio/stripes/components';
+
+class SaveButton extends React.Component {
+  static propTypes = {
+    pristine: PropTypes.bool,
+    submitting: PropTypes.bool,
+    stripes: PropTypes.shape({
+      hasPerm: PropTypes.func.isRequired,
+    }).isRequired,
+  };
+
+  render() {
+    const {
+      pristine,
+      submitting,
+      stripes,
+    } = this.props;
+    const dataTest = omit(this.props, ['pristine', 'submitting', 'stripes']);
+
+    return (
+      <Button
+        {...dataTest}
+        type="submit"
+        buttonStyle="primary paneHeaderNewButton"
+        marginBottom0
+        disabled={pristine || submitting || !stripes.hasPerm('settings.oai-pmh.edit')}
```

Comment:
I would suggest passing just one `disabled` prop here from outside. This will decrease the number of props which are required here.

Reply:
Fixed